src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsInformation.tsx

```diff
@@ -31,18 +32,26 @@ export const SettingsEditSettingsInformation: React.FC<
   const { sendToast } = useToasts()
   const { submitMutation } = useUpdateSettingsInformation()
   const phoneNumber = me.phoneNumber?.display ?? me.phoneNumber?.originalNumber
-  const phoneCountryCode = me.phoneNumber?.regionCode ?? "us"
+  const phoneCountryCode = me.phoneNumber?.regionCode
+
+  const countryInputOptions = sortCountriesForCountryInput(countryPhoneOptions)
+  const locationBasedInitialValues =
+    useInitialLocationValues(countryInputOptions)
+  const defaultPhoneCountryCode = phoneCountryCode
+    ? phoneCountryCode
+    : locationBasedInitialValues.phoneNumberCountryCode || "us"
 
   return (
     <>
       <Text variant={["md", "lg"]} mb={4}>
         Information
       </Text>
       <Formik
+        key={`formik-${defaultPhoneCountryCode}`}
```

Comment:
@rquartararo : I had to do this to be able to see the location updated in the country code. I guess this fix could also be added to the Pick-up, too. Though, I'm not super happy with the solution, so if you have a better suggestion, I'm all 👂s.

Reply:
I chatted with @rquartararo in our 1:1 and I learned she used the same solution for the pick-up.